sunpy/data/test/let/Fe_sectored_ahead_2015_01_10min_level1_11.txt

```diff
@@ -0,0 +1,43 @@
+2015 001.027778 00 40 00 -9999.90 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03
+2015 001.034722 00 50 00 -9999.90 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03
+2015 001.041667 01 00 00 -9999.90 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03
+2015 001.048611 01 10 00 -9999.90 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03
+2015 001.055556 01 20 00 -9999.90 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03
```

Comment:
Well the entire file has this (-9999 or -9.9999E+03) (invaild) or zeroes (valid). I have added rows with zeroes.